### PR TITLE
[ko] Update outdated files in dev-1.26.-ko.1 [M149-155]

### PR DESCRIPTION
**Pull request by Virusuki**
Related Issue: #[38458](https://github.com/kubernetes/website/issues/38458#issuecomment-1512426915)

Updated outdated files in content/en/docs/tasks/.

> ### 7 files to be modified
- [ ] M149. content/en/docs/tasks/administer-cluster/cluster-upgrade.md | 9(+XS) 0(-)
- [ ] M150. content/en/docs/tasks/administer-cluster/dns-custom-nameservers.md | 59(+M) 112(-)
- [ ] M151. content/en/docs/tasks/administer-cluster/kubeadm/kubeadm-upgrade.md | 4(+XS) 4(-)
- [ ] M152. content/en/docs/tasks/administer-cluster/manage-resources/memory-default-namespace.md | 8(+XS) 0(-)
- [ ] M153. content/en/docs/tasks/administer-cluster/network-policy-provider/cilium-network-policy.md | 26(+S) 30(-)
- [ ] M154. content/en/docs/tasks/administer-cluster/use-cascading-deletion.md | 1(+XS) 141(-)
- [ ] M155. content/en/docs/tasks/configmap-secret/managing-secret-using-config-file.md | 55(+M) 1(-)

### CODE REVIEW

**Review by jongwooo — CHANGES_REQUESTED** [review]
안녕하세요! 몇 가지 코멘트를 추가해보았습니다. 살펴보시고, 적절히 적용해주시면 감사하겠습니다.

File: content/ko/docs/tasks/administer-cluster/dns-custom-nameservers.md [hunk 1]
```diff
@@ -167,88 +190,12 @@ data:
     }
 ```
 
-`Kubeadm` 툴은 kube-dns 컨피그맵에서 동일한 설정의 CoreDNS 컨피그맵으로의
-자동 변환을 지원한다.
-
 {{< note >}}
-kube-dns는 스텁 도메인 및 네임서버(예: ns.foo.com)에 대한 FQDN을 허용하지만 CoreDNS에서는 이 기능을 지원하지 않는다.
+CoreDNS는 스텁 도메인 및 네임서버(예: ns.foo.com)에 대한 FQDN를 지원하지 않는다.
```
[the remark this answers is not in the document]
Comment (Virusuki) [reply content/ko/docs/tasks/administer-cluster/dns-custom-nameservers.md:194]: 리뷰 감사합니다. ㅎㅎ

File: content/ko/docs/tasks/administer-cluster/network-policy-provider/cilium-network-policy.md [hunk 2]
```diff
@@ -41,45 +41,40 @@ minikube version: v1.5.2
...
-실리움은 클러스터 구성을 자동으로 감지하고 
-성공적인 설치를 위해 적절한 구성 요소를 설치한다.
+minikube의 경우 CLI 도구를 사용하여 실리움을 설치할 수 있다. 그러기 위해서, 
+먼저 다음과 같은 명령을 사용하여 최신 버전의 CLI를 다운로드 한다.
 
 ```shell
 curl -LO https://github.com/cilium/cilium-cli/releases/latest/download/cilium-linux-amd64.tar.gz
+```
+
+이후 다음과 같은 명령을 사용하여 다운받은 파일을 `/usr/local/bin` 디렉토리에 압축푼다.
```
[the remark this answers is not in the document]
Comment (Virusuki) [reply content/ko/docs/tasks/administer-cluster/network-policy-provider/cilium-network-policy.md:51]: 리뷰 감사합니다. ㅎㅎ

File: content/ko/docs/tasks/configmap-secret/managing-secret-using-config-file.md [hunk 3]
```diff
@@ -170,6 +170,61 @@ type: Opaque
 
 `YWRtaW5pc3RyYXRvcg==`는 `administrator`으로 디코딩된다.
 
+## 시크릿(Secret) 작성 {#edit-secret}
+
+매니페스트를 사용하여 생성한 시크릿의 데이터를 편집하려면, 매니페스트에서 `data`
+및 `stringData` 필드를 수정하고 해당 파일을 클러스터에 적용합니다.
```
[the remark this answers is not in the document]
Comment (Virusuki) [reply content/ko/docs/tasks/configmap-secret/managing-secret-using-config-file.md:176]: 리뷰 감사합니다. ㅎㅎ

**Comment by Virusuki** [comment]
@jongwooo 목적격 조사와 놓친 평어체 부분 수정했습니다. 
리뷰 감사합니다~ㅎㅎ

**Comment by ysyukr** [comment]
@Virusuki, @jongwooo 확인이 늦었습니다.
/lgtm
/approve

**Comment by Virusuki** [comment]
네 확인감사합니다. 👍

**Pull request merged**